ironfish-rust/src/serializing/mod.rs

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+pub mod aead;
+
+use crate::errors::IronfishError;
+
+/// Helper functions to convert pairing parts to bytes
+///
+/// The traits in the pairing and zcash_primitives libraries
+/// all have functions for serializing, but their interface
+/// can be a bit clunky if you're just working with bytearrays.
+use ff::PrimeField;
+use group::GroupEncoding;
+
+use std::io;
+
+const HEX_CHARS: &[u8; 16] = b"0123456789abcdef";
+
+pub(crate) fn read_scalar<F: PrimeField, R: io::Read>(mut reader: R) -> Result<F, IronfishError> {
```

Comment:
`read_scalar` and `read_point` are unchanged, git just didn't catch that this was name changed from `serializing.rs`.